### ISSUE TITLE
Accessibility: Menus not draggable over VoiceOver

### ISSUE THREAD

**Issue by iamgabrielma** [wordpress-mobile/WordPress-iOS#17121]
Reported on 4271850-zen

> I'm blind and I'm using the iOS app for my iPhone. I created a website and then three pages such as Apps, Home and Skills and then a menu, but I can't select the positions. Can't find the button for the position either

### Expected behavior

In theory we can double tap on a menu item and this activates the menu to be draggable. I assume this is the expected behaviour by the VoiceOver speech.

### Actual behavior

You need to double tap and then keep the finger on top of the menu so it becomes draggable, still this happens maybe 1 every 10 times, the rest of the times you'll only hear a "blip" sound and the menu cannot be dragged around.

### Steps to reproduce the behavior
- In your phone go to Settings > Accessibility > turn on VoiceOver.
- In your WordPress app, go to Menus
- Attempt to move menus around via drag and drop

**Comment by twstokes** [comment]
👋 @iamgabrielma and @startuptester. I looked into this and it seems like we're offering a similar experience to the Mail App. In VoiceOver mode the rows can't be reordered unless the "handle" is selected.

I do think we could improve the experience by announcing the order (I like how the Mail App says "before X" or "after Y"), but after watching these videos do you think we still have a bug? Thanks!

|Mail App|WPiOS|
|---------|------|
| <video src="https://user-images.githubusercontent.com/2092798/157156322-98e97056-f6b2-49db-9592-1de2f00b262f.mp4" /> | <video src="https://user-images.githubusercontent.com/2092798/157156363-6eb218c4-6029-4479-ae3d-ea86dab3a221.mp4" /> |
